geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/PropertyManagement.java

```diff
@@ -138,7 +139,7 @@ public Serializable get(
 		try {
 			convertedValue = converter.convert(value);
 		}
-		catch (Exception e) {
+		catch (final Exception e) {
```

Comment:
If we re-throw an new exception we should pass the old exception to the constructor of the new one so the stack trace is maintained (looks like it wasn't changed in this pull request - just noticed it here)